README.md

```diff
@@ -109,6 +109,53 @@ Instead of `v-model` you can use the `index` prop and `change` event:
 />
 ```
 
+### Events
+
+`TinyBox` also emits the following events:
+ - `onPrev`/`onNext` with the next image being shown as a param
+ - `onClose` when `TinyBox` is closed
```

Comment:
I think we would be better off naming the events `prev`, `next`, and `close`. The "@" symbol already represents "on", just like in `@click="doStuff();"`

Reply:
Yeah you right, sorry coming from a Java/Kotlin background we usually have the `on*` prefix with these actions. Will change